Classes/Private/Manager/Audio/OCTAudioEngine.h

```diff
@@ -9,17 +9,23 @@
 #import <Foundation/Foundation.h>
 #import "OCTToxAV.h"
 
-typedef NS_ENUM(NSInteger, OCTAudioScope) {
-    OCTInput,
-    OCTOutput,
-};
-
 @interface OCTAudioEngine : NSObject
 
 @property (weak, nonatomic) OCTToxAV *toxav;
 @property (nonatomic, assign) OCTToxFriendNumber friendNumber;
 
 /**
+ * YES to not send audio frames over to tox, otherwise No.
+ */
+@property (nonatomic, assign) BOOL enableMicrophone;
+
+/**
+ * Default initializer.
+ * @param error Pointer to error object.
+ */
+- (instancetype)init:(NSError **)error;
```

Comment:
Oh, I see. You want to setup it just once. Anyway, please create separate from `init` method that will setup engine. You can use `dispatch_once` to guarantee that setup will be done only once.